packages/react-table/src/demos/Table.md

```diff
@@ -2397,52 +2394,77 @@ ComposableTableSortable = () => {
-                    sort: {
-                      sortBy: {
-                        index: activeSortIndex,
-                        direction: activeSortDirection
-                      },
-                      onSort,
-                      columnIndex
-                    }
-                  };
-                  const infoParams =
-                    columnIndex === 2
-                      ? {
-                          info: {
-                            tooltip: 'More information'
-                          }
-                        }
-                      : {};
-                  return (
-                    <Th key={columnIndex} {...sortParams} {...infoParams}>
-                      {column}
-                    </Th>
-                  );
-                })}
-              </Tr>
-            </Thead>
-            <Tbody>
-              {rows.map((row, rowIndex) => (
-                <Tr key={rowIndex}>
-                  {row.map((cell, cellIndex) => (
-                    <Td key={`${rowIndex}_${cellIndex}`} dataLabel={columns[cellIndex]}>
-                      {cell}
-                    </Td>
-                  ))}
+        <PageSection
+          padding={{
+            default: 'noPadding',
+            xl: 'padding'
+          }}
+        >
+          <Card component="div">
+            {tableToolbar}
+            <TableComposable aria-label="Sortable Table">
+              <Thead>
+                <Tr>
+                  <Th />
+                  {columns.map((column, columnIndex) => {
+                    const sortParams = {
+                      sort: {
+                        sortBy: {
+                          index: activeSortIndex,
+                          direction: activeSortDirection
+                        },
+                        onSort,
+                        columnIndex
+                      }
+                    };
+                    return (
+                      <Th key={columnIndex} {...sortParams}>
+                        {column}
+                      </Th>
+                    );
+                  })}
                 </Tr>
-              ))}
-            </Tbody>
-          </TableComposable>
+              </Thead>
+              <Tbody>
+                {rows.map((row, rowIndex) => (
+                  <Tr key={rowIndex}>
+                    <>
+                      <Td
+                        select={{
+                          rowIndex,
+                          onSelect: (_event, isSelecting) => onSelectNode(row, rowIndex, isSelecting),
+                          isSelected: isNodeSelected(row)
+                        }}
+                      />
+                      <Td dataLabel={columns[0]}>
+                        <div>{row[0]}</div>
+                        <a href="#">siemur/test-space</a>
+                      </Td>
+                      <Td dataLabel={columns[1]}>
+                        <CodeBranchIcon key="icon" /> {row[1]}
```

Comment:
Just a nit, but looks like this whitepsace character between the icon and text is getting removed. Is there a way to preserve that? I suppose we could use `&nbsp;`, though not sure if there are downsides to that. We could also use a flex layout with an `xs` or `sm` spacer - I think an `sm` spacer looks better there anyways.

@mcarrano wdyt? This first cell has a small spacer between the icon and number after it. The other cells just use a whitespace (single space) character between the icon and number.

<img width="312" alt="Screen Shot 2022-05-11 at 5 16 27 PM" src="https://user-images.githubusercontent.com/35148959/167956596-4a35b55e-8f44-4a82-85ba-2937abed419a.png">

Reply:
Ah, I see what you mean @mcoker Which do you feel is more correct? Agree that these should be consistent. We talked about creating a common data set for all these demos so we shouldn't need to be implementing one-off fixes for things like this.

Reply:
@mcarrano since the core and react examples can handle whitespace differently, it's probably best practice to use a layout for these icons. It also gives us control over the spacing, alignment, etc. For example, we use a flex layout  for icons + text in the card demos

<img width="598" alt="Screen Shot 2022-05-12 at 9 05 25 AM" src="https://user-images.githubusercontent.com/35148959/168094587-4fdecb3e-2a0c-4642-ad8c-2284ba824240.png">

Reply:
Using a `Flex` layout here is a great idea @mcoker ! I've updated this code to wrap the row data and icons with `Flex` using a `sm` spacer as you suggested/preferred.

Reply:
It's so pretty now!